src/lib.rs

```diff
@@ -405,3 +411,42 @@ fn formatted_display(result: Result<IonCReaderHandle, IonCError>, mode: &FormatM
         }
     }
 }
+
+struct Mndlbrt;
```

Comment:
This is fine, but not idiomatic. 

I think what you're trying to do here is "namespace" or "group" some functions so that it's clear that CANVAS, LIMIT, plot and so on are all related to your Mandleplotting. That's a good goal, but the OOP way of associating functions to a class is not quite what you'd do in Rust.

Probably, you'd just shove these in a module. The way I'd do it is to put it in mandlebrot.rs.

Reply:
Yeah I suspected I was off pistee here. Will refactor. Easier to remove it that way :)